lcd/config_downloader.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+import uuid
+import requests
+
+class ConfigDownloader:
+    def __init__(self, config_path, remote_url):
+        self.config_path = config_path
+        self.remote_url  = remote_url
+        self.api_key      = uuid.uuid4()
+
+    def run(self):
+        print "Setting up a new account"
```

Comment:
yeah, for cli logging is no good. But I think I'll be moving the CLI to emonpiLCD so the CLI bit won't be used.